adapters/flipp/flipp_test.go

```diff
@@ -1,11 +1,17 @@
 package flipp
 
 import (
+	"encoding/json"
 	"testing"
+	"time"
 
+	"github.com/SirDataFR/iabtcfv2"
+	"github.com/aws/smithy-go/ptr"
```

Comment:
no need to use `github.com/aws/smithy-go/ptr` 
should use utils from openrtb package

refer - https://github.com/prebid/prebid-server/pull/3250/files#r1379756913

Reply:
fixed